clientv3/watch.go

```diff
@@ -616,7 +625,9 @@ func (w *watchGrpcStream) run() {
 					},
 				}
 				req := &pb.WatchRequest{RequestUnion: cr}
-				wc.Send(req)
+				if err := wc.Send(req); err != nil {
+					lg.Warningf("when sending request, encountered: %v", err)
```

Comment:
same here

Reply:
Do you want to include req ?
that seems to be verbose.

Reply:
Actually I just realized the caller information (line number and filename) is included in the log prints. So there is no need to use distinguishable error messages. Could you change all 5 error messages to something like `"error when sending request: %v", err`. Sorry for the back and forth.